config/build.sh

```diff
@@ -23,14 +23,21 @@ mv golangci-lint-${GOCILINT_VERSION}-linux-amd64/golangci-lint /usr/local/bin
 
 curl -L -o operator-sdk $OPERATOR_SDK_LOCATION
 echo ${OPERATOR_SDK_SHA256SUM} operator-sdk | sha256sum -c
+chmod ugo+x operator-sdk
 mv operator-sdk /usr/local/bin
 
 curl -L -o opm $OPM_LOCATION && \
 echo ${OPM_SHASUM} opm | sha256sum -c
+chmod ugo+x opm
 mv opm /usr/local/bin
 
 python3 -m pip install PyYAML==5.3.1
 
+# Per https://git-scm.com/download/linux, we have two choices for CentOS
+# (which is what we're running on):
+# - Build from source
+# - Use a third party repository
+# For security reasons, we're preferring the former.
 GIT_VERSION="2.28.0"
 GIT_SHASUM="02016d16dbce553699db5c9c04f6d13a3f50727c652061b7eb97a828d045e534"
 GIT_DEPENDENCIES="epel-release perl-CPAN gettext-devel perl-devel openssl-devel zlib-devel curl-devel expat-devel getopt asciidoc xmlto docbook2X"
```

Comment:
I think we need to break apart this variable into `BUILD_DEPENDENCIES` and `RUNTIME_DEPENDENCES`. When we remove `curl-devel` and `openssl-devel` we are removing core packages depended by ssh and other core utils.

Reply:
Per offline discussion, we determined these were sane build deps not affecting the real `curl` or `ssh`. So leaving this alone.